rules_ruby_gapic/private/ruby_gapic_library_internal.bzl

```diff
@@ -61,6 +61,7 @@ def ruby_gapic_library_internal(
         grpc_service_config,
         service_yaml,
         rest_numeric_enums,
+        transport,
```

Comment:
We missed a couple param comments above, please add the `transport`

Reply:
👍🏼